guides/source/action_text_overview.md

```diff
@@ -46,6 +46,8 @@ happens after every keystroke, and avoids the need to use execCommand at all.
 ## Installation
 
 Run `rails action_text:install` to add the Yarn package and copy over the necessary migration.
+Also you need to set up Active Storage to use Action Text to manage attachments.
```

Comment:
I noticed that we used the `embedded images and other attachments` elsewhere in the guide. So how about? 
`Also, you need to set up Active Storage for embedded images and other attachments.`

Also, it seems that an unrelated file(`.5_2_release_notes.md.swp`) have been included. Please remove ;)

Reply:
I agree with you that is to the point to use `embedded images and other attachments` instead of `manage attachments`.
I fixed it and removed unnecessary swap file.　Thank you for pointing it out!